### PR TITLE
Add support of php && fix small bug of fx.py

### PR DESCRIPTION
**Pull request by chlins**

### CODE REVIEW

**Comment by metrue** [comment]
@chlins awesome, would you be nice also update the README ?

File: images/php/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM php:latest
+
+COPY . .
+ESPOSE 3000
```
Comment (DaidoujiChen) [inline images/php/Dockerfile]: typo?

Comment (chlins) [reply images/php/Dockerfile]: Oh~~~i'm sorry

**Comment by chlins** [comment]
Ok

**Pull request merged**